go.mod

```diff
@@ -27,10 +27,10 @@ require (
 	github.com/stretchr/testify v1.7.1
 	github.com/urfave/cli v1.22.9
 	golang.org/x/exp v0.0.0-20220426173459-3bcf042a4bf5
-	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
-	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
-	google.golang.org/grpc v1.39.0
-	google.golang.org/protobuf v1.27.1
+	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
```

Comment:
Which version of Go are you running? Did you do a `go mod tidy` that triggered these changes?

Reply:
Figured that out, was on go1.19

Went to go1.18, and then these files were unmodified